include/irr/asset/filters/CSummedAreaTableImageFilter.h

```diff
@@ -130,6 +131,12 @@ class CSummedAreaTableImageFilter : public CMatchedSizeInOutImageFilterCommon, p
 			const auto arrayLayers = state->inImage->getCreationParameters().arrayLayers;
 			static constexpr auto maxChannels = 4u;
 
+			asset::DefaultSwizzle swizzleMapping;
+			{
+				const auto swizzle = state->swizzle;
+				swizzleMapping.swizzle = swizzle;
+			}
```

Comment:
why add a swizzle to SAT just because one example texture uses it, it makes far more sense to have SAT be oblivious to the swizzle and make sure you apply an inverse swizzle (Same swizzle) when saving the new/output texture to file.